public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
```

Comment:
Sure, I was intending to do it very simple and straight forward so I though an external style sheet was not needed since there are very few styles... but thinking in all the other fallback interfaces we will be doing maybe it makes sense to have it in a separate css.